lib/msf/core/exploit/remote/winrm.rb

```diff
@@ -233,6 +233,9 @@ def send_winrm_request(data, timeout=20)
       'password' => datastore['PASSWORD'],
       'ctype' => "application/soap+xml;charset=UTF-8"
     }
+    print_good(opts['username'])
+    print_good(opts['password'])
+    print_good(opts['uri'])
```

Comment:
If we're keeping these lines could you please add some context around them so users know what values they're looking at?

Reply:
Debugging code left in by accident - removed.